.esh/commands/reshape_i18n_keys.py

```
@@ -0,0 +1,394 @@
+
+	path_map = combine_maps(auto_map, extra_manual)
+	reconcile_extra_through_nest(path_map, extra_manual)
+
+	with open(zh_path, "w", encoding="utf-8", newline="\n") as handle:
+		handle.write(dumps_locale(zh_data))
+
+	for file_name in locale_files:
+		if file_name == "zh-CN.json":
+			continue
+		abs_path = os.path.join(LOCALES_DIR, file_name)
+		with open(abs_path, "r", encoding="utf-8") as handle:
```

Ruff (0.16.0)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)